### PR TITLE
SCSS

### PR DESCRIPTION
**Pull request by kikmedia**
## Description

Added SCSS files, removed unused *src.css files, switched support screen image url to https

## Checklist
- [X] Read and understood the [CONTRIBUTING guidelines](CONTRIBUTING.md)
- [X] Added myself to the `@authors` in touched PHP files
- [X] Checked the changes with phpcq and introduced no new issues

### CODE REVIEW

File: contao/assets/scss/style.scss [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+	> legend {
+		margin: 0;
+		padding: 10px 0 10px 23px;
+		background: $icon_filter_setting no-repeat left center;
+	}
+	legend label {
+		font-weight: bold;
+	}
+}
+body[class*="version_2"] {
```
Comment (discordier) [inline contao/assets/scss/style.scss]: I wonder if this is really still needed as we are Version 3/4 now.

Comment (zonky2) [reply contao/assets/scss/style.scss]: I think we can delete all `body[class*="version_N"] {`

Comment (kikmedia) [reply contao/assets/scss/style.scss]: You are absolutely right. I will update the PR tomorrow.

Comment (zonky2) [reply contao/assets/scss/style.scss]: ?

Comment (zonky2) [reply contao/assets/scss/style.scss]: @kikmedia which "tomorrow"?

Comment (discordier) [reply contao/assets/scss/style.scss]: @kikmedia ping?

Comment (zonky2) [reply contao/assets/scss/style.scss]: we wait a long time :(

Comment (zonky2) [reply contao/assets/scss/style.scss]: fixed in https://github.com/MetaModels/core/pull/1286

**Comment by zonky2** [comment]
all fixed in https://github.com/MetaModels/core/pull/1286

**Pull request closed without merging**